x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Cytoscape.tsx

```diff
@@ -169,6 +172,7 @@ export function Cytoscape({
       });
     };
     const mouseoverHandler: cytoscape.EventHandler = event => {
+      trackApmEvent({ metric: 'service_map_object_hover' });
```

Comment:
What's the "object" being hovered?

Reply:
Asking because I honestly don't know, and it'll be even harder to figure out when looking at this metric on the telemetry cluster.

Reply:
right now the only objects in cytoscape are nodes and edges, so we could rename this `service_map_node_or_edge_hover` to make it more clear.

Reply:
Ah okay! So it's whenever either of them are hovered this will trigger? Personally I'd definitely prefer the explicit version (`service_map_node_or_edge_hover`)

Reply:
https://github.com/elastic/kibana/pull/61178